lxqt-config-appearance/man/lxqt-config-appearance.1
.TH LXQT-CONFIG-APPEARANCE "1" "January 2025" "LXQt 2.1.0" "LXQt Appearance Settings"
.SH NAME
lxqt-config-appearance \- GUI appearance application of \fBLXQt\fR: The Lightweight
Qt Desktop Environment
.SH SYNOPSIS
.B lxqt-config-appearance
.br
.SH DESCRIPTION
With this application you can set and configure the Qt / LXQt GUI appearance. Options
include configuring widget style, color palette, application toolbar, icon theme,
LXQt theme, font, cursor, and optionally GTK 2/3 style.
.P
On Linux/Unix, the look of a GUI application depends on the toolkit it is based on.
For LXQt, the overall look of many apps relies on the Qt framework GUI. The
\fBlxqt-config-appearance\fR application provides all aspects of configuring the
appearance of the \fBLXQt\fR desktop.
.P
All the remaining major aspects of LXQt appearance rely on the active Qt widget
style because the \fBLXQt\fR DE is based on the Qt framework.
.P
.SH OPTIONS
-v, --version  Displays version information.
.br
-h, --help     Displays help on command line options.
.br
--help-all     Displays help, including generic Qt options.
.SH "REPORTING BUGS"
Report bugs to https://github.com/lxqt/lxqt-config/issues
.SH "SEE ALSO"
\fBLXQt\fR it has been tailored for users who value simplicity, speed, and
an intuitive interface. LXQt is also intended for less powerful machines. Also,
See:
.P
\fBlxqt-session(1)\fR  LXQt Session Manager
.br
\fBlxqt-config(1)\fR   LXQt Configuration Center


CHANGELOG
lxqt-config-2.1.0 / 2024-11-05
===============================
 * Fixed a FTBFS with Clang in `lxqt-config-input`.
 * Show unsupported message for `lxqt-config-input` under wayland.
 * Show unsupported message for `lxqt-config-monitor` under wlroots-based compositors.
 * Fixed a memory leak in `lxqt-config-input`.
 * Added nullity check for monitor config.
 * Safer size saving for the dialog of `lxqt-config-appearance`.
 * Do not load monitor or input settings under Wayland.
 * More informative tooltips in lxqt-config dialog.
 * Standardized desktop files for settings.
 * Consistent capitalization for GenericName for desktop files.
 * Added tooltip colors to LXQt Appearance.
 * Added checkbox to use palette provided by LXQt theme.
 * Fixed an old problem in setting wallpapers.
 * Don't start `xsettingsd` outside X11.
 * Added an option for toolbar icon size.
 * Make sure Monitor Settings dialog is shown completely on Wayland.
 * Remember size of Monitor Settings dialog.
 * Start `xsettingsd` on X11 only when GTK themes should be set.
 * Scale monitor images on resizing monitor settings dialog.

lxqt-config-2.0.0 / 2024-04-17
===============================
 * Ported to Qt6.
 * Improved the UI of monitor settings.
 * Corrected tab orders by reordering blocks of UI files.

lxqt-config-1.4.0 / 2023-11-05
===============================
 * Use lxqt-menu-data.
 * Fixed setting of primary monitor after a change in libkscreen.
 * Put cursor settings in ~/.Xresources (fixes the problem of not being able to set the cursor or its size when Xresources from `/etc` is read).
 * Corrected some typos.
 * Improved the message about default monitor settings.
 * Don't return uninitialized struct members.

lxqt-config-1.3.0 / 2023-04-15
===============================
 * Fixed potential segfault when loading keyboard layout settings.
 * lxqt-config-monitor: More header file inclusion for libkscreen 5.26.90.

lxqt-config-1.2.0 / 2022-11-05
===============================
 * Enhancements to theme list in appearance config dialog. Themes are always sorted alphabetically, user themes are distinguished, and theme folders can be opened by double clicking or by using context menus.

lxqt-config-1.1.0 / 2022-04-15
===============================
 * Updated `README.md`.
 * Fixed translation in "Remove Palettes" dialog of LXQt Appearance Configuration.
 * Removed Qt4 dead conditional code.
 * Removed unneeded KF5 KwindowSystem dependency.
 * Added a separate page for GTK style settings to LXQt Appearance Configuration.
 * Remember size of lxqt-config-input dialog.
 * Fixed manual editing of combos in LXQt Locale Configuration.
 * Fixed `mimeapps.list` path with empty `XDG_CURRENT_DESKTOP`.

lxqt-config-1.0.0 / 2021-11-04
===============================
 * Bumped minimum required Qt version to 5.15.
 * Do not force openbox configs under LXQt.
 * Made "Other Settings" work in Configuration Center.
 * Save and load Qt palettes.

lxqt-config-0.17.1 / 2021-04-16
===============================
  * Fixed compilation against Qt 5.12.

lxqt-config-0.17.0 / 2021-04-15
===============================
  * Dropped obsolete methods and fixed compiler warnings.
  * Completely moved to Qt5 signal/slot syntax.
  * Made lxqt-config-appearance work under Wayland.

lxqt-config-0.16.1 / 2020-11-14
===============================
  * Fixed a regression in setting the acceleration speed of mouse/touchpad.

lxqt-config-0.16.0 / 2020-11-01
===============================
  * Better UI for lxqt-config-brightness.
  * Avoid C++11 range for possible container detachment.
  * Only use QStringLiteral where it avoids allocations.
  * Initialize monitor settings config in "lxqt-config-monitor/monitor.h".
  * Use QFlags default constructor.
  * Fixed a crash in lxqt-config-monitor.
  * Fixed type associations and added "Default Application".
  * Detailed palette customization in lxqt-config-appearance.
  * Remember sizes of appearance and file associations dialogs.

lxqt-config-0.15.0 / 2020-04-22
===============================
  * Fixed changing of shortcut for switching keyboard layout.
  * Inserted default GTK3 themes in lxqt-config-appearance.
  * Set cursor style in GTK2, GTK3 and xsettingsd.
  * Don't use automatic string conversions.
  * C++11 code updates.
  * A better method to drag and drop monitors in lxqt-config-monitor.
  * Removed old mouse accel/sensitivity options.
  * Fixed warnings on applying settings in lxqt-config-monitor.
  * Fixed forgetting of lxqt-config-monitor's config.
  * Removed old mouse accel/sensitivity options.

lxqt-config-0.14.1 / 2019-02-24
===============================

  * Apply button for appearance and input config dialogs
  * Updated translations
  * Note for packagers: lxqt-config now depends on liblxqt >= 0.14.1

lxqt-config-0.14.0 / 2019-01-25
===============================

  * Implemented Features:
    - Introduced GTK appearance settings
    - Initial touchpad settings and improved input device handling
  * Implementation fixed:
    - lxqt-config: Removed ancient Qt workaround that creates problems now
    - lxqt-config-appearance: Some GUI fixes
    - lxqt-config-appearance: mkpath if settings of GTK doesn't exists.
    - lxqt-config-brightness: Backlight control updated
    - lxqt-config-brightness:
      XRandrBrightness::backlight_get_with_range return value
    - lxqt-config-monitor: Monitor name is added to output.
  * Improved cmake scripting
    - Set cmake_minimum_required to 3.1.0
    - Check for xi and libudev
    - Removed QUIET from find_package
    - Removed dependency on QtConcurrent
    - Removed locale compile definitions
    - Installed former missed man pages
  * Moved translations from lxqt-l10n back to lxqt-policykit
    - Removed obsolete translation fuctionality
    - Added translation promo in README.md
  * Translation updates

lxqt-config-0.13.0 / 2018-05-21
===============================

  * Spanish translation update
  * Update name for shortcut component

lxqt-config-appearance/translations/lxqt-config-appearance.ts
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="en_US">
<context>
    <name>ColorLabel</name>
    <message>
        <location filename="../colorLabel.cpp" line="37"/>
        <source>Click to change color.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../colorLabel.cpp" line="61"/>
        <source>Select Color</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>ConfigOtherToolKits</name>
    <message>
        <location filename="../configothertoolkits.cpp" line="178"/>
        <source>GTK themes</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../configothertoolkits.cpp" line="179"/>
        <source>&lt;p&gt;&apos;%1&apos; has been overwritten.&lt;/p&gt;&lt;p&gt;You can find a copy of your old settings in &apos;%2&apos;&lt;/p&gt;</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../configothertoolkits.cpp" line="244"/>
        <source>Error</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../configothertoolkits.cpp" line="244"/>
        <source>Error: gsettings cannot be run</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>FontsConfig</name>
    <message>
        <location filename="../fontsconfig.ui" line="23"/>
        <source>Font</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="30"/>
        <source>Default font for user interface</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="36"/>
        <source>Font name:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="46"/>
        <source>Style:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="77"/>
        <source>Point size:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="54"/>
        <source>Normal</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="59"/>
        <source>Bold</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="64"/>
        <source>Italic</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="69"/>
        <source>Bold Italic</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="94"/>
        <source>The following settings only affect newly started applications</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="100"/>
        <source>Use antialias fonts</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="153"/>
        <source>Font hinting style:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="118"/>
        <location filename="../fontsconfig.ui" line="164"/>
        <source>None</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="169"/>
        <source>Slight</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="174"/>
        <source>Medium</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="179"/>
        <source>Full</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="146"/>
        <source>Font hinting</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="187"/>
        <source>Resolution (DPI):</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="204"/>
        <source>Autohint</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="107"/>
        <source>Subpixel antialiasing:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="123"/>
        <source>RGB</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="128"/>
        <source>BGR</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="133"/>
        <source>VRGB</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="138"/>
        <source>VBGR</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>GTKConfig</name>
    <message>
        <location filename="../gtkconfig.ui" line="44"/>
        <source>GTK Style</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../gtkconfig.ui" line="63"/>
        <source>Set GTK themes (GTK configuration files will be overwritten!)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../gtkconfig.ui" line="93"/>
        <source>GTK 3 Theme</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../gtkconfig.ui" line="77"/>
        <source>GTK 2 Theme</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../gtkconfig.ui" line="103"/>
        <source>To attempt uniform theming, either select similar style/theme (if available) across all lists, or select &apos;gtk2&apos; Qt style (if available) to mimic GTK themes.

Make sure &apos;xsettingsd&apos; is installed to help GTK applications apply themes on the fly.</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>IconThemeConfig</name>
    <message>
        <location filename="../iconthemeconfig.ui" line="14"/>
        <source>LXQt Appearance Configuration</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../iconthemeconfig.ui" line="26"/>
        <source>Icons Theme</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../iconthemeconfig.ui" line="87"/>
        <source>The KDE extension of XDG icon themes -&gt; FollowsColorScheme</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../iconthemeconfig.ui" line="90"/>
        <source>Colorize icons based on widget style (palette)</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>LXQtThemeConfig</name>
    <message>
        <location filename="../lxqtthemeconfig.ui" line="23"/>
        <source>LXQt Theme</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../lxqtthemeconfig.ui" line="57"/>
        <source>Use wallpaper provided by theme</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../lxqtthemeconfig.ui" line="64"/>
        <source>Use palette provided by theme</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../lxqtthemeconfig.cpp" line="83"/>
        <source>(User Theme)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../lxqtthemeconfig.cpp" line="185"/>
        <source>Open theme folder</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>PalettesDialog</name>
    <message>
        <location filename="../palettes.ui" line="14"/>
        <location filename="../palettes.ui" line="20"/>
        <source>Palettes</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../palettes.ui" line="33"/>
        <source>Filter...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../palettes.ui" line="43"/>
        <source>&amp;Remove</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>QObject</name>
    <message>
        <location filename="../main.cpp" line="61"/>
        <source>LXQt Appearance Configuration</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../main.cpp" line="74"/>
        <source>Widget Style</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../main.cpp" line="83"/>
        <source>Icons Theme</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../main.cpp" line="92"/>
        <source>LXQt Theme</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../main.cpp" line="100"/>
        <source>Font</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../main.cpp" line="110"/>
        <source>Cursor</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../main.cpp" line="117"/>
        <source>GTK Style</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>StyleConfig</name>
    <message>
        <location filename="../styleconfig.ui" line="33"/>
        <source>Widget Style</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="63"/>
        <source>Qt Palette</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="77"/>
        <source>Some Qt styles may ignore these colors.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="102"/>
        <source>Window:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="112"/>
        <source>View:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="122"/>
        <source>Selection:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="132"/>
        <source>Tooltip:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="146"/>
        <source>Link:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="166"/>
        <source>Window Text:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="176"/>
        <source>View Text:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="186"/>
        <source>Selected Text:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="196"/>
        <source>Tooltip Text:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="210"/>
        <source>Visited Link:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="242"/>
        <source>&amp;Save Palette</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="249"/>
        <source>&amp;Load Palette</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="269"/>
        <source>&amp;Default Palette</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="284"/>
        <location filename="../styleconfig.ui" line="294"/>
        <location filename="../styleconfig.ui" line="326"/>
        <location filename="../styleconfig.ui" line="336"/>
        <location filename="../styleconfig.ui" line="368"/>
        <source>May be overridden by Qt style or app</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="287"/>
        <source>Toolbar button style:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="298"/>
        <source>Only display the icon</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="303"/>
        <source>Only display the text</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="308"/>
        <source>The text appears beside the icon</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="313"/>
        <source>The text appears under the icon</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="318"/>
        <source>Default</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="329"/>
        <source>Toolbar icon size:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="340"/>
        <source>16</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="345"/>
        <source>22</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="350"/>
        <source>24</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="355"/>
        <source>32</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="360"/>
        <source>48</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="371"/>
        <source>Activate item on single click</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="51"/>
        <source>Qt Style</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.cpp" line="330"/>
        <location filename="../styleconfig.cpp" line="340"/>
        <source>Save Palette</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.cpp" line="330"/>
        <source>Palette name:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.cpp" line="341"/>
        <source>A palette with the same name exists.
Do you want to replace it?</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.cpp" line="484"/>
        <source>Remove Palettes</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.cpp" line="485"/>
        <source>Do you really want to remove selected palette(s)?
Root palettes will remain intact if existing.</source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>
